unlock-app/src/components/interface/locks/Create/elements/CreateLockFormSummary.tsx

```diff
@@ -195,13 +195,13 @@ export const CreateLockFormSummary = ({
             <span className="text-xl font-bold">{formData?.name}</span>
```

Comment:
`On the lock deployment UI: let's add more context. Instead of "Name", let's have "That's the max you can airdrop for this lock to a single address."`

@julien51 you mean as description under here?